src/bignum.rs

```diff
@@ -0,0 +1,39 @@
+//! The bignum system library.
+
+use super::*;
+
+mod native {
+    extern "C" {
+        pub fn bignum_mul256(a: *const u32, b: *const u32, ret: *mut u32);
+        pub fn bignum_umulmod256(a: *const u32, b: *const u32, modulo: *const u32, ret: *mut u32);
+    }
+}
+
+pub fn mul256(a: &Uint256, b: &Uint256) -> Uint256 {
+    let mut ret = Uin256::default();
```

Comment:
Typo here. `Uint256::default();`

Reply:
Why does this compile?!

Reply:
Well, because the feature was disabled. See #56.